docs/tutoriales/tutorial-react.md

```diff
@@ -0,0 +1,237 @@
+>[Más Información](https://es.reactjs.org/)
+
+## Herramientas 
+
+Para React se sugieren las siguientes herramientas:
+
+* Navegador Web (Google Chorme, Moxilla Firefox)
+* [Node Js](https://nodejs.org/es/)
+* Administradores de Paquetes ([Brew](https://brew.sh/index_es), [Chocolatey](https://chocolatey.org/) )
+* Terminal, se puede utilizar con la terminal nativa pero te recomendamos [iTerm2](https://iterm2.com/) , [Hyper](https://hyper.is/) , [cmder](https://cmder.net/)
+* Editor de código, se sugiere que se utilice [Visual Studio Code](https://code.visualstudio.com/)
+
+## Extensiones
+
+Estas extensiones son una recomendación para mejorar tu experiencia en el desarrollo con React.
+
+* Instalar [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=es), esta se encuentra en la chrome web store (Navegador Web)
+* Debugger for Chrome (Visual Studio Code)
+* React Extension Pack (Visual Studio Code)
+* Auto Close Tag (Visual Studio Code)
+* Prettier - Code Formatter (Visual Studio Code)
+
+
+* * *
+## Verificar Instalaciones
+
+Verificar Node js, con estos comandos se puede verificar la versión y además si están instalados.
+
+```shell
+$ node -v
+#v14.15.2
+$ npm -v
+#6.14.9
+```
+
+## Iniciar la primera aplicación con React
+
+Vamos a crear una primera aplicación con React en donde se van a ver reflejados algunas de las funcionalidades de React, en este caso debes cambiar "nombre_de_tu_aplicación" por el nombre que desees, para hacer más sencilla la lectura de ahora en adelante se utilizara my-app.
+
+```shell
+$ npx create-react-app "nombre_de_tu_aplicación"
+```
+Ahora debes entrar a tu directorio de my-app y arrancar tu aplicación.
+
+```shell
+$ cd my-app
+$ yarn start
+```
+Para ver tu código debes utilizar, duplicando tu terminal, lo siguiente: 
+
+```shell
+$ code .
+```
+
+## Primeros Pasos
+
+Una vez abierto el código en tu editor, en este caso recomendablemente Visual Studio Code, puedes navegar hasta encontrar dentro de la carpeta *src* el archivo `App.js` con un contenido similar a este:
+```javascript
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Editar <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
+
+export default App;
+```
+Puedes cambiar el contenido por el siguiente y hacer los cambios que desees como primeros pasos para tu aventura en React. Además, puedes ver los cambios en tu navegador, para esto debes mantener tu terminal con el servidor abierto.
+
+```javascript
+import { Component } from 'react';
```

Comment:
Group imports by types

**First:**
External Libraries

**Second:**
Internal components, utils, config files etc

**third:**
parent files (such styles or child components)

and always add a blank space between them

example:

```
import { Component } from 'react';

import logo from './logo.svg';
import './App.css';
```